website/docs/r/sfn_state_machine.html.markdown

```diff
@@ -209,6 +209,27 @@ This resource exports the following attributes in addition to the arguments abov
 
 ## Import
 
+In Terraform v1.12.0 and later, the [`import` block](https://developer.hashicorp.com/terraform/language/import) can be used with the `identity` attribute. For example:
+
+```terraform
+import {
+  to = aws_sfn_state_machine.example
+  identity = {
+    "arn" = "arn:aws:states:us-east-1:123456789012:stateMachine:MyWorkflow-2024-09-17-a7b3c8f1"
```

Comment:
For consistency we can match the value used in the examples for other import methods.

```suggestion
    "arn" = "arn:aws:states:eu-west-1:123456789098:stateMachine:bar"
```